core/src/main/java/org/mskcc/cbio/oncokb/util/CacheUtils.java

```diff
@@ -529,23 +511,14 @@ private static void synAlterations() {
         }
     }
 
-    public static void updateGene(Integer entrezGeneId) {
-        try {
-            System.out.println("Update gene on instance " + PropertiesUtils.getProperties("app.name") + " at " + MainUtils.getCurrentTime());
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
-
-        System.out.println(entrezGeneId.toString() + " is queued.");
-        genesToBeUpdated.add(entrezGeneId);
-        notifyOtherServices("update", entrezGeneId);
-    }
-
     public static void forceUpdateGeneAlterations(Integer entrezGeneId) {
         alterations.remove(entrezGeneId);
     }
 
-    public static void updateGene(Integer entrezGeneId, Boolean propagate) {
+    public static void updateGene(Integer entrezGeneId, Boolean propagate, Boolean skipGeneUpdateDelay) {
```

Comment:
Looks like skipGeneUpdateDelay is never used in any implementation.

Reply:
Ah yes, let me clean it up